src/main/java/org/calrissian/flowmix/api/aggregator/LongSumAggregator.java

```diff
@@ -32,6 +32,7 @@
 import static java.lang.System.currentTimeMillis;
 import static java.util.UUID.randomUUID;
 
+@Deprecated
 public class LongSumAggregator implements Aggregator {
```

Comment:
I think the types should be broken out in some way where a user can specify a smaller numeric type if they know they will never need it to be a long. This was one of the reasons for breaking out a  different aggregator just for type- though I never got to the next layer of abstraction. There could be millions of aggregators sitting in memory at any point in time, 8 vs 4 bytes could be a huge deal if the user knows they will never need it.